docs/source/strategies/collaborative_training_basic.rst

```diff
@@ -0,0 +1,43 @@
+:orphan:
+
+.. _collaborative_training_basic:
+
+Training on unreliable mixed GPUs across the internet (Basic)
+=============================================================
+
+Collaborative Training tries to solve the need for top-tier multi-GPU servers by allowing you to train across unreliable machines,
+such as local machines or even preemptible cloud compute across the internet.
+
+Under the hood, we use `Hivemind <https://github.com/learning-at-home/hivemind>`_ which provides de-centralized training across the internet.
+
+To use Collaborative Training, you need to first install Hivemind.
+
+.. code-block:: bash
+
+    pip install hivemind
+
+The ``CollaborativeStrategy`` accumulates gradients from all processes that are collaborating till they reach a ``target_batch_size``. By default, we use the batch size
+of the first batch to determine what each local machine batch contributes towards the ``target_batch_size``. Once the ``target_batch_size`` is reached, an optimizer step
+is made on all processes.
+
+.. warning::
+
+    When using ``CollaborativeStrategy`` note that you cannot use gradient accumulation (``accumulate_grad_batches``). This is because hivemind manages accumulation internally.
```

Comment:
```suggestion
    When using ``CollaborativeStrategy`` note that you cannot use gradient accumulation (``accumulate_grad_batches``). This is because Hivemind manages accumulation internally.
```